### DISCUSSION TITLE
Something was wrong in Archinstall

### DISCUSSION THREAD

**Discussion by zhanhere** [archlinux/archinstall#1639] [Q&A]
I don't know what happened,and I only can take a picture.
![image](https://user-images.githubusercontent.com/90076318/218240170-3d0d1158-04e0-40c8-8dd0-827f5911d11a.jpeg)
How to use archinstall correctly？

After I asked the question,I tried to install Debian. The same way in Debian it's ok. it looks like maybe the wrong in Archinstall.
I tried to install Debian is in graphic.

I forget write what I did...
I partitioned the hard drive only.

**Comment by TheRootDaemon** [comment]
Hey, it looks like you have problems with mounting or with the filesystem state.

- The partitions may not be properly formatted with filesystems yet
- There could be existing filesystems that need to be unmounted first
- The EFI system partition might not be properly set up
- There may be issues with the partition table or disk state

Use `lsblk` or `fdisk -l` to see the current partition layout, format if needed.

Or you could use the `best effort partitioning` in the archinstall script that would handle all the partitioning and all the other stuff.
But that would erase all the existing partitions and rewrite it with whatever you choose ext4 or btrfs or whatever it is.

**Comment by svartkanin** [comment]
You'll need to provide the log file otherwise debugging will be challenging
